archiver.go

```diff
@@ -172,6 +177,34 @@ func (deb *ArchiveDeb) RelaxedUnpack(destination string) error {
 	return deb.Unpack(destination)
 }
 
+// Helper function for packing with external tool
+func pack(command []string) error {
+	return Command{}.Run("Pack", command...)
+}
+
+func (arc *ArchiveBase) Pack(destination string) error {
+	return fmt.Errorf("Pack is not supported for '%s'", arc.file)
+}
+
+func (tar *ArchiveTar) Pack(destination string) error {
+	command := []string{"tar"}
+	command = append(command, "-c")
+
+	if compression, ok := tar.options["tarcompression"]; ok {
+		if unpackTarOpt := tarOptions(compression.(string)); len(unpackTarOpt) > 0 {
+			command = append(command, unpackTarOpt)
+		} else {
+			// defaults to "gz"
+			command = append(command, "-z")
```

Comment:
This doesn't default to gz; this uses gz as a fallback e.g. if compression for some reason gets set to lzo4 you end up  compressing as gz.